pkg/version/version_test.go

```diff
@@ -190,8 +190,8 @@ func TestGetChainsVersionViaConfigMap(t *testing.T) {
 			configMap:             getConfigMapData("chains-info", "test", map[string]string{"app.kubernetes.io/part-of": "tekton-chains"}),
 			want:                  "test",
 		}, {
-			name:      "get chains version from configmap in tekton-chains namespace",
-			namespace: "tekton-chains",
+			name:      "get chains version from configmap in tekton-pipelines namespace",
+			namespace: "tekton-pipelines",
```

Comment:
any reason for changing this?
`tekton-chains` is already there in `defaultNamespaces` list so it should have checked there

Reply:
fixed